### PR TITLE
Add about page

### PR DESCRIPTION
**Pull request by tinykite**
## What this does 
Adds an about page, plus many supporting elements. 

## Summary 
- Adds `about.html`, plus a web component template, `cap-about-page`
- Adds `cap-anchor-list` component, for the sidebar links shown on many interior pages
- Adds `cap-page-header`, for all color-blocked interior page headers
- Adds `cap-contributor-list` component, for the many past and present contributors to CAP
- Updates `cap-decorated-header` to support h1 or h2 headings, multiple sizes of heading, different decorators, 
- Refined styling for `cap-media-list`
- Renamed `cap-header` to `cap-nav` to avoid confusion with page header components
- Added quite a large number of root element styles, utility classes, one layout class, and unscoped component css styles  for adhoc styling on interior pages. If you're interested about the strategy behind this, you can read more in the Harry Roberts article [BEMIT: taking the BEM naming convention a step further](https://csswizardry.com/2015/08/bemit-taking-the-bem-naming-convention-a-step-further/).
  -  Utility classes are prefixed `u`
  -  Layout classes  are prefixed `l` 
  - Component styles are prefixed `c`
 - Refined all global responsive design styles, across components shared between the about page and the homepage and root/global css selectors
 - Added three new font-size css variables
 - Added supporting JavaScript files containing structured data for the about sidebar links as well as each contributor to CAP
 - Removed unused tooltip JavaScript
 
## Screenshots
Desktop view: 

Screenshot 2023-12-20 at 4 05 45 PM

Smaller mobile view: 
Screenshot 2023-12-20 at 4 05 58 PM

## How to test
- Add a new remote for this fork 
- Checkout this branch
- Spin up your favorite webserver (the default one, used by the VSCode-suggested server plugin, will spin up the homepage at `https://localhost:5501/`
- Visit https://localhost:5501/about.html — note that the ".html" is crucial!

### CODE REVIEW

File: components/cap-contributor-list.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		css`
+			:host {
+				--color-link: var(--color-blue-400);
+				--color-link-hover: var(--color-gray-600);
+				--color-publisher: var(--color-gray-500);
+				--color-date: var(--color-gray-400);
+				--list-style-type: none;
+
+				font-family: var(--font-sans-text);
+			}
```
Comment (tinykite) [inline components/cap-contributor-list.js:22]: Super cool thing about css variables — you can define them locally in `:root` (for a page) or `:host` (for scoped web components) and then reuse that variable anywhere else within that scoped component or page, or later reassign them depending on your UI theming needs.

File: css/components.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	}
+}
+
+.c-article * + * {
+	margin-block-start: var(--spacing-200);
+}
+
+.c-article h2 + * {
+	margin-block-start: var(--spacing-100);
+}
```
Comment (tinykite) [inline css/components.css:41]: I typically try to write CSS with a flat structure, and often using BEM — for example, `.articleList` and `.articleList__content` but sometimes more specific classes are convenient, for example adding a margin between all children elements of an `article` element.

**Comment by matteocargnelutti** [comment]
Thank you for asking me to have a look, @tinykite! 

This looks great 🏄 ! 

Only unimportant suggestion I have: since  `<cap-about>` doesn't contain logic, I think it can be implemented directly in `about.html`, like you did for `index.html`?

Cheers

**Review by matteocargnelutti — APPROVED** [review]

**Comment by tinykite** [comment]
> Thank you for asking me to have a look, @tinykite!
> 
> This looks great 🏄 !
> 
> Only unimportant suggestion I have: since `<cap-about>` doesn't contain logic, I think it can be implemented directly in `about.html`, like you did for `index.html`?
> 
> Cheers

Thank you! And yes, that's a very good question! After thinking it through, I felt like it might be easiest for developer experience to do this, so that I could easily import JSON into the template itself without a data fetch. 

I think I could have alternatively done something like this in a plain html file, inside of a script block: 

```
import anchorLinks from "../data/aboutSidebarLinks.js" assert {type: 'json'};
```

But if caniuse data is correct, that `assert` — to avoid an error message like is still experimental and not fully supported by all makor browsers. I think this is necessary to avoid a browser error like: 

```Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "application/json". Strict MIME type checking is enforced for module scripts per HTML spec.```

Which would happen because JSON is considered separate than a module. But I am admittedly not well-read on the difference between JSON and modules.

**Comment by matteocargnelutti** [comment]
@tinykite This makes a lot of sense 👍 !

**Pull request merged**